src/components/Overlay/OverlayTiles/OverlayTile.tsx

```diff
@@ -4,28 +4,27 @@ import styled from 'styled-components';
 import Icon from '../../Icons';
 import { CollaborationItem } from '../../../assets/content';
 import SmallParagraph from '../../SmallParagraph';
+import { SlackButton } from '../../SlackButton';
+import Paragraph from '../../Paragraph';
 
 const StyledTileWrapper = styled.div`
   display: flex;
   align-items: flex-start;
-  justify-content: center;
-
-  @media screen and (max-width: ${p => p.theme.screens.tablet}) {
-    flex-direction: column;
-  }
+  flex-direction: column;
 `;
 
 const StyledContentWrapper = styled.div`
   display: flex;
   flex-direction: column;
   align-items: start;
   justify-content: center;
+  font-size: 1rem;
 `;
 
 const StyledTileTitle = styled.h2`
-  font-size: ${p => p.theme.fontSizeL};
+  font-size: 1rem;
```

Comment:
A bit nit-picky but don't we maintain the theme for not setting font sizes in random locations